commerce-theme-minium/commerce-theme-minium-impl/src/main/resources/META-INF/resources/render/view.jsp

```diff
@@ -58,11 +61,18 @@ String galleryId = PortalUtil.generateRandomKey(request, "gallery");
 
 				<c:choose>
 					<c:when test="<%= cpSku != null %>">
-						<div class="availability"><%= cpContentHelper.getAvailabilityLabel(request) %></div>
+						<div class="availability">
+							<c:if test='<%= stockQuantityLabel.equals("0") %>'>
+								<%= availabilityClassModifier = "bad" %>
+							</c:if>
 
-						<div class="availabilityEstimate"><%= cpContentHelper.getAvailabilityEstimateLabel(request) %></div>
+							<span class="commerce-dot commerce-dot--big commerce-dot--<%= availabilityClassModifier %>"></span>
+							<span>
+								<span class="availability__label"><%= availability %></span><span class="stockQuantity">| <%= stockQuantityLabel %></span>
+							</span>
+						</div>
 
-						<div class="stockQuantity"><%= cpContentHelper.getStockQuantityLabel(request) %></div>
+						<div class="availabilityEstimate"><%= cpContentHelper.getAvailabilityEstimateLabel(request) %></div>
```

Comment:
@gianmarcobrunialti This logic is wrong. Can you take out this change from the PR and send in a different one with the proper fix? So we can start merging the price fix

Reply:
@marco-leo My bad. I thought I had taken out those changes from the commit, but I must have committed them by mistake, sorry. Will take them off right away tomorrow morning.